### DISCUSSION TITLE
Windows 10 Taskbar Style on Windows 11

### DISCUSSION THREAD

**Discussion by Spartan2401** [valinet/ExplorerPatcher#3931] [Q&A]
Hi! I'm on Windows 11 version 24H2, and I'm new to the OS and Explorerpatcher overall.

There's a taskbar toolbar (Audio Band, awesome way to see and control the music currently being played) I came to rely on a lot in Windows 10. That app requires Explorepatcher or Startallback (paid) for Windows 11 since taskbar toolbars aren't a thing anymore on this OS.

I installed EP, but I'm unable to enable to enable the Windows 10 taskbar style in the EP Properties menu (screenshot below). Since there's an error message and everything ("Windows 10 taskbar is not available in this version of Windows), this is obviously a known issue. However, some Audio Band users on Win11 23H2 and 24H2 are having the same compatibility issue as me, while others with the same OS versions report they are still able to switch to the Win10 taskbar style.

I've tried unlocking and locking the taskbar (now called "Automatically hide the taskbar") and toggling other taskbar behavior settings, but nothing has worked. Are there any steps I can take to make the Win10 taskbar feature work on my version of Win11?

![image](https://github.com/user-attachments/assets/0fd793b6-4825-4677-9524-d37ff6788fed)

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

If you don't see that option, you're running an older version of EP.  Update to the latest version of EP: https://github.com/valinet/ExplorerPatcher/releases

**Comment by Joe-48041** [comment]
Install Explorer Patcher version ending with 4317.67.1 or newer from:
https://github.com/valinet/ExplorerPatcher/releases
Press CTRL-Shift ESC to open task manager.
Scroll all of the way down to bottom.
Right Click "Windows Explorer" and click "Restart".

Now right click taskbar and click "properties".
A new windows opens.
Notice on the left side is Taskbar in bold and underlined.
To the right of that, click on "Taskbar style *:Windows 10"
There should be 3 choices:
  Windows 11 (default)
  Windows 10
  Windows 10 (ExplorerPatcher)
Click the bottom one, "Windows 10 (ExplorerPatcher)"

In the very bottom left corner, click on "Restart File Explorer (*)"

You should now be able to right click the taskbar and unlock it.
Then drag the taskbar to the left or right side.
Then right click the taskbar and lock it.

**Reply by plazer1** [reply]
I had to manually download a newer version, because updating directly from the EP Properties window did nothing.

**Reply by Mehdixxxn** [reply]
its work
